### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1074] [General]
Does someone know about a piece of software that can bulk convert FIT to GPX?
thank you

**Comment by FJBDev** [comment]
A quick [google search ](https://www.google.com/search?client=firefox-b-d&q=bulk+convert+fit+to+gpx)lead me to [this](https://trailgazing.wordpress.com/2018/07/19/batch-convert-fit-file-to-gpx-in-windows/)

**Reply by gringojack** [reply]
Thanks for the clue
